docs/porting/configuration-differences.md

```diff
@@ -0,0 +1,597 @@
+  </td>
+  <td>
+  <p><span>OFF</span></p>
+  </td>
+  <td>
+  <p><span>Configurable</span></p>
+  </td>
+ </tr>
+ <tr>
+  <td>
+  <p><span>X509_CHECK_FLAG_NEVER_CHECK_SUBJECT</span></p>
+  </td>
+  <td>
+  <p><span>OFF</span></p>
+  </td>
+  <td>
+  <p><span>Configurable</span></p>
+  </td>
+ </tr>
+ <tr>
+  <td>
+  <p><span>X509_CHECK_FLAG_ALWAYS_CHECK_SUBJECT</span></p>
+  </td>
+  <td>
+  <p><span>OFF</span></p>
+  </td>
+  <td>
+  <p><span>NO-OP</span></p>
+  </td>
+ </tr>
+ <tr>
+  <td>
+  <p><span>X509_CHECK_FLAG_NO_PARTIAL_WILDCARDS</span></p>
+  </td>
+  <td>
+  <p><span>ON</span></p>
+  </td>
+  <td>
+  <p><span>NO-OP</span></p>
+  </td>
+ </tr>
+ <tr>
+  <td>
+  <p><span>X509_CHECK_FLAG_MULTI_LABEL_WILDCARDS</span></p>
+  </td>
+  <td>
+  <p><span>OFF</span></p>
+  </td>
+  <td>
+  <p><span>NO-OP</span></p>
+  </td>
+ </tr>
+ <tr>
+  <td>
+  <p><span>X509_CHECK_FLAG_SINGLE_LABEL_SUBDOMAINS</span></p>
+  </td>
+  <td>
+  <p><span>OFF</span></p>
+  </td>
+  <td>
+  <p><span>NO-OP</span></p>
+  </td>
+ </tr>
+ <tr>
+  <td rowspan=8>
+  <p>
+    <span>
+      <a href="https://www.openssl.org/docs/manmaster/man3/PKCS7_sign.html">
+        PKCS7_sign
+      </a>
+    </span>
+  </p>
+  </td>
+  <td>
+  <p><span>PKCS7_DETACHED</span></p>
+  </td>
+  <td>
+  <p><span>OFF</span></p>
+  </td>
+  <td>
+  <p><span>Configurable</span></p>
+  </td>
+ </tr>
+ <tr>
+  <td>
+  <p><span>PKCS7_BINARY</span></p>
+  </td>
+  <td rowspan=3>
+  <p><i><span>Partially Supported<br>
+  </span></i><sp><br>
+  These flags must be used simultaneously together with </span>
+  <span style='font-family:"Courier New"'>PKCS7_DETACHED</span>
+  <span> to generate a detached RSA
+  SHA-256 signature of the data and produces a PKCS#7 SignedData structure
+  containing it.</span></p>
+  </td>
+  <td rowspan=3>
+  <p><b><span>Must be used along with </span></b><b>
+        <span>PKCS7_DETACHED</span></b>
+  <span>. Other combinations are not
+  supported<b>.</b></span></p>
+  </td>
+ </tr>
+ <tr>
+  <td>
+  <p><span>PKCS7_NOATTR</span></p>
+  </td>
+ </tr>
+ <tr>
+  <td>
+  <p><span>PKCS7_PARTIAL</span></p>
+  </td>
+ </tr>
+ <tr>
+  <td>
+  <p><span>PKCS7_TEXT</span></p>
+  </td>
+  <td>
+  <p><span>OFF</span></p>
+  </td>
+  <td>
+  <p><span>NO-OP</span></p>
+  </td>
+ </tr>
+ <tr>
+  <td>
+  <p><span>PKCS7_NOCERTS</span></p>
+  </td>
+  <td>
+  <p><span>OFF</span></p>
+  </td>
+  <td>
+  <p><span>NO-OP</span></p>
+  </td>
+ </tr>
+ <tr>
+  <td>
+  <p><span>PKCS7_STREAM</span></p>
+  </td>
+  <td>
+  <p><span>OFF</span></p>
+  </td>
+  <td>
+  <p><span>NO-OP</span></p>
+  </td>
+ </tr>
+ <tr>
+  <td>
+  <p><span>PKCS7_NOSMIMECAP</span></p>
+  </td>
+  <td>
+  <p><span>OFF</span></p>
+  </td>
+  <td>
+  <p><span>NO-OP</span></p>
+  </td>
+ </tr>
+ <tr>
+  <td rowspan=4>
+  <p><span>EVP_PKEY_assign</span></p>
+  </td>
+  <td>
+  <p><span>EVP_PKEY_DH</span></p>
+  </td>
+  <td>
+  <p><span>Not Supported</span></p>
+  </td>
+  <td>
+  <p><span>NO-OP</span></p>
```

Comment:
Should the "Not Supported" Flags have "N/A" instead of NO-OP in Configurability?

Reply:
No strong opinions here, but I wanted to make the used wording consistent with what was in the text.